.appveyor-mingw.yml

```diff
@@ -0,0 +1,18 @@
+version: 1.0.{build}

```

Comment:
You would have to add an extra appveyor build for this. I have done that in an attempt to shorten the overall build time. But I have an idea on how I can put it in one file and still keep the overall build time low.